droid-help/src/main/resources/Web pages/Whats New.html

```diff
@@ -64,18 +64,39 @@ <h5 align="center">
         </td>
       </tr>
     </table>
+    <p>
+      Droid 6.6.1 offers the following fixes and improvements:
+    </p>
+    <ul>
+      <li>#898 Bugfix: Use backslash on Windows OS in no profile output</li>
+      <li>#900 Bugfix: Failure to consistently output HASH column in CSV</li>
+      <li>Updated to use latest version of Java 11 bundled JRE</li>
+      <li>#886 Bugfix: Terminology corrected on the preferences dialog</li>    </ul>
     <p>
       Droid 6.6.0 offers the following new features:
     </p>
     <ul>
-      <li>Internal API for making droid easier to use as a dependency</li>
-      <li>Updated bundled jre to Java 11</li>
+      <li>Updated to use Java 11 as bundled JRE</li>
```

Comment:
we were already using Java 11 in 6.5.2 (11.0.5) but we bring up to a newer JRE version in 6.6.0 and then the latest current in 6.6.1.

This should probably just say "updated JRE to newer version" against 6.6.0

Reply:
done